### PR TITLE
fix(pipeline-service-account) copy Service account while rerunning Pipelines

### PR DESCRIPTION
**Pull request by abhinandan13jan**
Addresses issue:
https://jira.coreos.com/browse/ODC-1501
--------------------------------------------------------------------------
POST-FIX
--------------------------------------------------------------------------
rerunning pipeline
Screenshot from 2019-07-31 18-46-18
-------------------------------------------------------------------------
pipelinerun running
Screenshot from 2019-07-31 18-46-30
-------------------------------------------------------------------------
pipelinerun successful
Screenshot from 2019-07-31 18-46-46
-------------------------------------------------------------------------
serviceAccount of pipelineRun
Screenshot from 2019-07-31 18-47-02
--------------------------------------------------------------------------

### CODE REVIEW

File: frontend/packages/dev-console/src/utils/pipeline-actions.tsx [hunk 1]
```diff
@@ -104,6 +104,12 @@ export const newPipelineRun = (pipeline: Pipeline, latestRun: PipelineRun): Pipe
       trigger: {
         type: 'manual',
       },
+      serviceAccount:
+        latestRun && latestRun.spec && latestRun.spec.serviceAccount
+          ? latestRun.spec.serviceAccount
+          : pipeline && pipeline.spec && pipeline.spec.serviceAccount
```
Comment (christianvogt) [inline frontend/packages/dev-console/src/utils/pipeline-actions.tsx]: Trying to find where in the docs it says that a `pipeline` has a `serviceAccount` property. I cannot find. Please confirm if this is correct.

Comment (abhinandan13jan) [reply frontend/packages/dev-console/src/utils/pipeline-actions.tsx]: As there is no creation flow in place for `Pipeline`, if `resources`, `params` and `serviceAccount` is added to the `Pipeline` YAML in `Spec`, all the data should cascade down to every corresponding `PipelineRuns` when `Start Pipeline/rerun Pipeline` is triggered.  
Is there any other option to specifically borrow these properties to pass on to the first-pipelinerun for a pipeline if not present in the Pipeline itself? @siamaksade @vdemeester  @sspeiche

**Comment by christianvogt** [comment]
/lgtm
/approve

We do not support the start scenario yet and can address the `serviceAccount` later for that scenario.

**Pull request merged**